cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
+    let customView = PaymentView()
+    let payment: Payment?
+
+    func callbacks() {
+        view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
```

Comment:
you should follow `Single Responsibility Principle` and separate this line to another method
```
func setupUI() { 
    view.backgroundColor = UIColor(displayP3Red: 1, green: 1, blue: 1, alpha: 1)
}
```